### PR TITLE
TE-241 Video widget

### PR DESCRIPTION
**Pull request by davidpelayo**
[Related YouTrack issue](https://youtrack.lodgify.net/issue/TE-241)

### What **one** thing does this PR do?
- Defines the Video widget.

### CODE REVIEW

File: src/components/widgets/Video/component.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+Component.defaultProps = {
+  isUrl: false,
+  videoInput: null,
+};
+
+Component.propTypes = {
+  /** Whether the input is an url or not
+   */
+  isUrl: PropTypes.bool,
```
Comment (marcdavi-es) [inline src/components/widgets/Video/component.js]: I don't think we need this prop if we're validating the string?

If it's a url, do the url dance, no?

File: src/components/widgets/Video/component.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  /** The string that will be used to build the video player. It could be
+   *  whether a URL or a embeddable HTML snippet.
+   */
+  videoSource: PropTypes.string,
```
Comment (marcdavi-es) [inline src/components/widgets/Video/component.js]: `videoInput`

File: src/components/widgets/Video/utils/getVideoContent.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    return (
+      <div
+        className="video is-html"
+        dangerouslySetInnerHTML={{ __html: DOMPurify.sanitize(videoInput) }}
+      />
+    );
+  }
+
+  // Otherwise
+  return null;
```
Comment (marcdavi-es) [inline src/components/widgets/Video/utils/getVideoContent.js]: I don't know if this should fail silently. Perhaps we should throw an error if the `videoInput` is not valid.

Comment (davidpelayo) [reply src/components/widgets/Video/utils/getVideoContent.js]: I wouldn't get now into error handling. But surely this is not a good pattern.

Comment (davidpelayo) [reply src/components/widgets/Video/utils/getVideoContent.js]: Anyway, makes sense. I will throw an error instead.

File: src/components/widgets/Video/component.js (hunk 1 above)
Comment (marcdavi-es) [reply src/components/widgets/Video/component.js]: Ah I see. I think it's OK though. If the consumer is passing a bad string, they can eat the performance hit 🍔

File: src/components/widgets/Video/component.js (hunk 2 above)
Comment (davidpelayo) [reply src/components/widgets/Video/component.js]: I renamed all to `videoSource`. Do you think is more understandable `videoInput`?

Comment (marcdavi-es) [reply src/components/widgets/Video/component.js]: I prefer `videoSource`

Comment (marcdavi-es) [reply src/components/widgets/Video/component.js]: but when I looked before they were inconsistent?

Comment (davidpelayo) [reply src/components/widgets/Video/component.js]: Yes, now they are fixed.

**Pull request merged**